.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
     - conda update --all --yes
     - travis_retry conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt
     - source activate TEST
-    - travis_retry conda install --yes --file requirements-dev.txt
+    - travis_retry pip install -r requirements-dev.txt
```

Comment:
OK -- I guess wait and see if they diverge at some point, and deal with it then.